samples/cellular/nrf_cloud_multi_service/src/cloud_connection.c

```diff
@@ -489,6 +490,14 @@ static int setup_cloud(void)
 		return err;
 	}
 
+	err = nrf_cloud_client_id_get(client_id, sizeof(client_id));
+	if (err) {
+		LOG_ERR("nRF Cloud client_id is not initialized, error: %d", err);
+		return err;
+	} else {
+		LOG_INF("nRF Cloud client_id is %s", client_id);
+	}
```

Comment:
In that case I think we should add this as a feature to the library itself rather than include it here

But sure, let's include it here and I will write a ticket to make printing the client ID as an INFO and remove this later

Reply:
Ticket exists now.

IRIS-6931